aptos-move/framework/src/natives/cryptography/bulletproofs.rs

```diff
@@ -171,24 +314,13 @@ fn verify_range_proof(
     bit_length: usize,
     dst: Vec<u8>,
 ) -> SafeNativeResult<SmallVec<[Value; 1]>> {
-    context.charge(
-        BULLETPROOFS_BASE
-            + BULLETPROOFS_PER_BYTE_RANGEPROOF_DESERIALIZE
-                * NumBytes::new(proof_bytes.len() as u64),
-    )?;
+    charge_gas(context, 1, bit_length)?;
```

Comment:
TODO: revert

touching this without feature gating will lead to execution divergence...